roles/calico/tasks/main.yml

```diff
@@ -4,7 +4,7 @@
 - name: Calico | Run kube proxy
   import_role:
     name: kube_proxy_and_dns
-  when: "ansible_hostname == groups.oo_first_master.0"
+  when: "ansible_nodename == groups.oo_first_master.0"
```

Comment:
nodename may not match ansible inventory hostname. The most correct would be `inventory_hostname`

Reply:
Thank you. It has been changed to inventory_hostname